File: 프로그래머스/1/77884. 약수의 개수와 덧셈/약수의 개수와 덧셈.java
class Solution {
    public int solution(int left, int right) {
        int answer = 0;
        for(int i = left; i <= right; i++){
            int s = (int) Math.sqrt(i);

            if(s * s == i) answer -= i;
            else answer += i;
        }
        return answer;
    }
}

File: 프로그래머스/1/77884. 약수의 개수와 덧셈/약수의 개수와 덧셈.js
function solution(left, right) {
    let answer = 0;
    for(let i = left; i <= right; i++){
        let s = Math.floor(Math.sqrt(i));
        
        if(s * s == i) answer -= i;
        else answer += i;
    }
    return answer;
}